content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,408 @@
+[Pod Security](/docs/concepts/security/pod-security-admission/)
+admission can be easily combined with admission webhooks and external services.
+
+Pod Security admission `restricted` policy, the most restrictive policy of the
+[Pod Security Standards](/docs/concepts/security/pod-security-standards/) set,
+[can operate in several modes](/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces),
+`warn`, `audit` or `enforce` to gradually apply the most appropriate
+[security context](/docs/tasks/configure-pod-container/security-context/)
+according to security best practices. Nevertheless, pods'
+[security context](/docs/tasks/configure-pod-container/security-context/)
+should be separately investigated to limit the privileges and access pods may
+have on top of the predefined security standards, for specific use cases.
+
+For a hands-on tutorial on [Pod Security](/docs/concepts/security/pod-security-admission/),
+see the blog post
+[Kubernetes 1.23: Pod Security Graduates to Beta](/blog/2021/12/09/pod-security-admission-beta/).
+
+[Memory and CPU limits](/docs/concepts/configuration/manage-resources-containers/)
+should be set in order to restrict the memory and CPU resources a pod can
+consume on a node, and therefore prevent potential DoS attacks from malicious or
+breached workloads. Such policy can be enforced by an admission controller.
+Please note that CPU limits will throttle usage and thus can have unintended
+effects on auto-scaling features or efficiency i.e. running the process in best
+effort with the CPU resource available.
+
+{{< caution >}}
+Memory limit superior to request can expose the whole node to OOM issues.
+{{< /caution >}}
+
+### Enabling Seccomp
+
+Seccomp can improve the security of your workloads by reducing the Linux kernel
+syscall attack surface available inside containers. The seccomp filter mode
+leverages BPF to create an allow or deny list of specific syscalls, named
+profiles. Those seccomp profiles can be enabled on individual workloads,
+[a security tutorial is available](/docs/tutorials/security/seccomp/). In
+addition, the [Kubernetes Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator)
+is a project to facilitate the management and use of seccomp in clusters.
+
+For historical context, please note that Docker has been using
+[a default seccomp profile](https://docs.docker.com/engine/security/seccomp/)
+to only allow a restricted set of syscalls since 2016 from
+[Docker Engine 1.10](https://www.docker.com/blog/docker-engine-1-10-security/),
+but Kubernetes is still not confining workloads by default. The default seccomp
+profile can be found [in containerd](https://github.com/containerd/containerd/blob/main/contrib/seccomp/seccomp_default.go)
+as well. Fortunately, [Seccomp Default](/blog/2021/08/25/seccomp-default/), a
+new alpha feature to use a default seccomp profile for all workloads can now be
+enabled and tested.
```

Comment:
```suggestion
as well.
```

and then in the What's Next section, link to https://k8s.io/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads